nixos/modules/services/matrix/lk-jwt-service.nix

```
@@ -0,0 +1,84 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.lk-jwt-service;
+in
+{
+  meta.maintainers = [ lib.maintainers.quadradical ];
+  options.services.lk-jwt-service = {
+    enable = lib.mkEnableOption "Enable lk-jwt-service";
+    package = lib.mkPackageOption pkgs "lk-jwt-service" { };
+
+    livekit = {
+      url = lib.mkOption {
+        type = lib.types.str;
```

```suggestion
        type = lib.types.strMatching "^wss?://.*";
```

We can validate the proto like this.

---

Fixed.